server/views/pages/contacts/manage/contactSearch.njk

```diff
@@ -40,96 +43,184 @@
+                    }
+                %}
+                
+                {% set contactNameHtml %}
+                    <a href="#" class="govuk-link govuk-link--no-visited-state bapv-result-row">
+                        {{ (result.lastName + ", " + result.firstName) | convertToTitleCase }}
+                    </a>
+                {% endset %}
+
+                {% set rows = (rows.push([
+                    { html: contactNameHtml },
+                    { text: result.dateOfBirth | formatDate('dd/MM/yyyy') },
+                    { html: convertAddressToLines(address) }
+                ]), rows) %}
+            {% else %}
+                {% set rows = (rows.push([
+                    {
+                        text: 'No results found',
+                        attributes: { colspan:3 },
+                        classes: 'govuk-!-text-align-centre'
+                    }
+                ]), rows) %}
+            {% endfor %}
+
+            {{ govukTable({
+                head: [
+                    { text: "Name" },
+                    { text: "Date of birth" },
+                    { text: "Address" }
+                ],
+                rows: rows
+            }) }}
+
+            {% if results.totalPages > 1 %}
+                {% set paginationItems = [] %}
+                {% for i in range(0, results.totalPages) %}
+                    {% if i === 0 or i === (results.totalPages - 1) or (i >= results.number - 1 and i <= results.number + 1) %}
+                        {% set paginationItems = (paginationItems.push(
+                            {
+                                number: i + 1,
+                                href: "?page=" + i,
+                                current: (i === results.number)
+                            }
+                        ), paginationItems) %}
+                        {% elif i === 1 or i === results.totalPages - 2 %}
+                        {% set paginationItems = (paginationItems.push({ ellipsis: true }), paginationItems) %}
+                    {% endif %}
+                {% endfor %}
+
+                {{ govukPagination({
+                    previous: {
+                        href: "?page=" + (results.number - 1)
+                    } if not results.first,
+                    next: {
+                        href: "?page=" + (results.number + 1)
+                    } if not results.last,
+                    items: paginationItems
                 }) }}
+            {% endif %}
+        </div>
 
-                <div class="govuk-button-group">
-                    {{ govukButton({
-                        html: "Search",
-                        type: "submit",
-                        classes: 'govuk-!-margin-top-6',
-                        attributes: {"data-qa": "search-button"}
-                    }) }}
-                </div>
-            </form>
+        <div class="govuk-grid-column-full govuk-!-margin-top-4">
+            {% if not view.isContactListed %}
```

Comment:
So we hide the link if any contact first, last or middle name is an exact match  (case-insensitive) ? That doesn't seem right.

Reply:
>CTA  “The contact is not listed” - takes user to “Contact confirmation” page (covered in PRC-47: Contact confirmation
>CTA will not be displayed until search is performed (this is effectively route into a new journey “Create new contact”)

I think the link is supposed to be displayed once any search is done. With that said, the AC is conflicting as it says it takes you to contact confirmation (wrong, that's for linking a contact) and then "Create new contact" (right, because we don't want to select a contact and instead create a new one)

Reply:
Changed as per AC

Reply:
Changes added as per AC